sql/2023/01/critical-css-opportunity.sql

```diff
@@ -0,0 +1,35 @@
+# HTTP Archive query to get % of WordPress sites not having Critical CSS implementation.
+#
+# WPP Research, Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See query results here: https://github.com/GoogleChromeLabs/wpp-research/pull/16
+SELECT
+  client,
+  total_wp_sites,
+  sites_with_critical_css,
+  (total_wp_sites-sites_with_critical_css) AS sites_without_critical_css,
+  CONCAT(ROUND((total_wp_sites-sites_with_critical_css)*100/total_wp_sites, 3),' %') AS opportunity
```

Comment:
Please avoid calculating a percentage for these query results, that is unnecessary. Let's stick with values like `0.3` instead of `30%` - we can easily format such values as percentages when using software like Excel or Google Sheets, no need to format the raw data.
```suggestion
  (total_wp_sites - sites_with_critical_css) / total_wp_sites AS opportunity
```